packages/flutter/lib/src/widgets/app.dart

```diff
@@ -341,7 +341,7 @@ class WidgetsApp extends StatefulWidget {
     this.showSemanticsDebugger = false,
     this.debugShowWidgetInspector = false,
     this.debugShowCheckedModeBanner = true,
-    this.inspectorSelectButtonBuilder,
+    this.exitWidgetSelectionButtonBuilder,
```

Comment:
this is a breaking change, so we'll need to follow the flutter breaking change policy for this

Reply:
Pinged on Flutter discord to get guidance on the process for this

Reply:
It seems this isn't considered a breaking change: https://discord.com/channels/608014603317936148/608018585025118217/1306392838224875621

But we should probably announce it after it has landed

Reply:
This broke https://pub.dev/packages/wiredash. Even if it is not considered a breaking change by the flutter policy, it still breaks user code.
Adding a deprecation next time would have been appreciated